homeworks/Natalia.Hanguga_NataHanguga/homework3/arrayFocus.js

```diff
@@ -0,0 +1,17 @@
+function createArray() {
+    const arr = [];
+    let arr1 = [];
+    const max = 32;
+    const step = 4;
+    for (let n = 0; n < max; n += step) {
+        arr.push(n);
+        console.log(n);
+    }
+    console.log(arr);
+    arr1 = arr.map(el => el ** 2);
+    console.log(arr1);
+    const sum = arr1.reduce((a, b) => a + b);
+    console.log(sum);
+}
+
+createArray();
```

Comment:
cannot find task 1.1

can you please divide all task in separate units (functions/loops)? very hard to read